### PR TITLE
Fixes some inaccurate messages which happen when you have less than two hands/feet

### PR DESCRIPTION
**Pull request by unid15**
-Sink, display case and tatorpeels now respect your amount of limbs and use proper singular/plural forms when referring to them

-Also new getHandAmount() and getFeetAmount() procs

100% tested

### CODE REVIEW

**Comment by dylanstrategie** [comment]
Please try to avoid writing things like if(!left_foot || left_foot.status & ORGAN_DESTROYED) .--

Return to the line and indent it correctly, it makes the code much clearer.

Also spaces before and after any mathematical logic, except for divisions. So . = 2

In the display case, path the user directly as [user], then use a macro (\his). Their should really be used in case of a major problem. Change "looks" to "stares"

Expanding the contraction in clowngrenade.dm (it's and they're) will make the sentence look a tad better, especially since feet/foot is here

File: code/game/objects/structures/displaycase.dm [hunk 1]
```diff
@@ -260,8 +260,19 @@
...
+			if(ishuman(user))
+				user.delayNextAttack(10) // prevent spam
+				var/mob/living/carbon/human/H = user
+				var/hands = H.getHandAmount()
 
+				if(hands >= 2)
+					user.visible_message("[user] gently runs \his hands over \the [src] in appreciation of its contents.", \
+						"You gently run your hands over \the [src] in appreciation of its contents.", \
+						"You hear someone streaking glass with their greasy hands.")
+				else if(hands == 1)
```
Comment (clusterfack) [inline code/game/objects/structures/displaycase.dm]: Just eliminate this center code block and use the [L.getHandAmount()>=2 ? "hands" : "hand"] in the top one

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -611,6 +611,32 @@
 	if(!.) 				. = if_no_id	//to prevent null-names making the mob unclickable
 	return
 
+/mob/living/carbon/human/getFeetAmount()
+	. = 2 //probably should use a define but whatever
+
+	var/datum/organ/external/left_foot = get_organ("l_foot")
+	var/datum/organ/external/right_foot = get_organ("r_foot")
+
+	if(!left_foot || left_foot.status & ORGAN_DESTROYED)
```
Comment (clusterfack) [inline code/modules/mob/living/carbon/human/human.dm]: I would put parenthesis here to be safe

File: code/modules/mob/living/living.dm [hunk 3]
```diff
@@ -302,6 +302,11 @@
 
 // ++++ROCKDTBEN++++ MOB PROCS //END
 
+/mob/living/proc/getFeetAmount()
```
Comment (clusterfack) [inline code/modules/mob/living/living.dm]: There are a few other mob types that would be different than this too, like larva

File: code/modules/mob/living/simple_animal/hostile/hivebot.dm [hunk 4]
```diff
@@ -59,6 +59,15 @@
 	del src
 	return
 
+//Taken from the sprite
+
+/mob/living/simple_animal/hostile/hivebot/getFeetAmount()
+	return 4
+
+/mob/living/simple_animal/hostile/russian/getHandAmount()
```
Comment (wild-billy) [inline code/modules/mob/living/simple_animal/hostile/hivebot.dm]: typo, should be hivebot not russian

File: code/game/objects/structures/watercloset.dm [hunk 5]
```diff
@@ -381,19 +381,29 @@
 		M << "<span class='warning'>Someone's already washing here.</span>"
 		return
 
-	usr << "<span class='notice'>You start washing your hands.</span>"
+	var/mob/living/L = M
+	if(!L) return
+
```
Comment (clusterfack) [inline code/game/objects/structures/watercloset.dm]: Looks like it needs an escape case for no hands

**Comment by clusterfack** [comment]
Besides ^ and squashums, looks good

File: code/game/objects/structures/watercloset.dm [hunk 6]
```diff
@@ -381,19 +381,29 @@
...
 
-	if(!Adjacent(M)) return		//Person has moved away from the sink
+	if(!Adjacent(L)) return		//Person has moved away from the sink
 
-	M.clean_blood()
-	if(ishuman(M))
-		M:update_inv_gloves()
-	for(var/mob/V in viewers(src, null))
-		V.show_message("<span class='notice'>[M] washes their hands using \the [src].</span>")
+	L.clean_blood()
```
Comment (clusterfack) [inline code/game/objects/structures/watercloset.dm]: Oh also shouldn't this just be cleaning blood from your hands, not the whole mob

File: code/modules/mob/living/silicon/mommi/mommi.dm [hunk 7]
```diff
@@ -40,6 +40,12 @@ They can only use one tool at a time, they can't choose modules, and they have 1
 /mob/living/carbon/can_use_hands()
 	return 1
 
+/mob/living/silicon/robot/mommi/getFeetAmount()
+	return 4
+
+/mob/living/silicon/robot/mommi/getHandAmount()
+	return 2
```
Comment (N3X15) [inline code/modules/mob/living/silicon/mommi/mommi.dm]: Should be 1

Comment (PJB3005) [inline code/modules/mob/living/silicon/mommi/mommi.dm]: Well, on the *sprites* the ones with actual hands have 2 hands.

Comment (N3X15) [inline code/modules/mob/living/silicon/mommi/mommi.dm]: The implementation has one, and the original crab sprite had a tool arm and a claw on the other.

Comment (PJB3005) [inline code/modules/mob/living/silicon/mommi/mommi.dm]: ah

**Comment by N3X15** [comment]
Conflicts

**Pull request closed without merging**